src/System.IO/src/System.IO.csproj

```diff
@@ -79,6 +84,11 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard13aot'">
+    <TargetingPackReference Include="System.Private.Interop" />
+    <ValidateIgnoreReference Include="System.Private.Interop" />
```

Comment:
This can be generalized to `<ValidateIgnoreReference Inlcude="@(TargetingPackReference)" />` I know it is only one in this case but it might be better to generalize this. If we start getting more of these libraries we should actually add this in a common target in BuildTools.